services/namex-pay/src/namex_pay/worker.py

```diff
@@ -25,6 +25,8 @@
 the model to a standalone SQLAlchemy usage with an async engine would need
 to be pursued.
 """
+import asyncio
+import nest_asyncio
```

Comment:
What's the new library for?

Reply:
Quoting https://github.com/erdewit/nest_asyncio: 
`By design asyncio does not allow its event loop to be nested. This presents a practical problem: When in an environment where the event loop is already running it's impossible to run tasks and wait for the result. Trying to do so will give the error "RuntimeError: This event loop is already running".`

That's the only way I could make it work for namex-pay.

Reply:
but why have you added the namex-api queue to the this listener, and in it's own loop.
solving a problem that maybe shouldn't have been created?

Reply:
Well, there's a problem. Maybe there's a better way to solve it:

Without adding the queue to namex-pay we got:
```
    raise err
  File "/home/andre/repos/namex/services/namex-pay/venv/lib/python3.8/site-packages/namex/services/queue.py", line 131, in publish_json_to_subject_sync
    self.loop.run_until_complete(self.async_publish_json(payload, subject))
AttributeError: 'NoneType' object has no attribute 'run_until_complete'

```

With:
`queue.init_app(FLASK_APP)`

```
  File "/home/andre/repos/namex/services/namex-pay/venv/lib/python3.8/site-packages/namex/services/queue.py", line 131, in publish_json_to_subject_sync
    self.loop.run_until_complete(self.async_publish_json(payload, subject))
  File "/usr/lib/python3.8/asyncio/base_events.py", line 592, in run_until_complete
    self._check_running()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 552, in _check_running
    raise RuntimeError('This event loop is already running')
RuntimeError: This event loop is already running
```

What do you suggest for this case?